src/components/app.tsx

```diff
@@ -41,7 +41,7 @@ export default function App() {
 				<MainSidebar className="basis-52 flex-shrink-0 h-full" />
 				<main
 					data-testid="site-content"
-					className="py-8 pr-8 bg-white overflow-y-auto h-full flex-grow rounded-chrome"
+					className="bg-white h-full flex-grow rounded-chrome overflow-hidden"
```

Comment:
These padding updates to the other tabs are to accommodate the Assistant tab having a different background color than the other three tabs. The general main window padding has been moved off the `<main />` window element to the individual tabs themselves.